Application/Sources/UI/Controllers/PageContainerViewController.swift

```diff
@@ -51,99 +47,42 @@ class PageContainerViewController: UIViewController {
-            pageView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
-            pageView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
-            pageView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
-        ])
-        
-        pageViewController.didMove(toParent: self)
-        
-        var hasImage = false
-        var tabBarItems = [UITabBarItem]()
-        viewControllers.forEach { viewController in
-            if let tabBarItem = viewController.tabBarItem {
-                tabBarItems.append(tabBarItem)
-                if tabBarItem.image != nil {
-                    hasImage = true
-                }
-            }
-        }
-        
-        let tabBar = MDCTabBar()
-        tabBar.itemAppearance = hasImage ? .images : .titles
-        tabBar.alignment = .center
-        tabBar.delegate = self
-        tabBar.items = tabBarItems
+        let tabView = tabContainerViewController.view!
+        view.insertSubview(tabView, at: 0)
         
-        tabBar.tintColor = .white
-        tabBar.unselectedItemTintColor = .srgGray96
-        tabBar.selectedItemTintColor = .white
-        
-        // Use ripple effect without color, so that there is no Material-like highlighting (we are NOT adopting Material)
-        tabBar.enableRippleBehavior = true
-        tabBar.rippleColor = .clear
-        
-        view.addSubview(tabBar)
-        self.tabBar = tabBar
-        
-        tabBar.translatesAutoresizingMaskIntoConstraints = false
-        tabBarTopConstraint = tabBar.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor)
-        NSLayoutConstraint.activate([
-            tabBarTopConstraint,
-            tabBar.heightAnchor.constraint(equalToConstant: 60.0),
-            tabBar.leadingAnchor.constraint(equalTo: view.leadingAnchor),
-            tabBar.trailingAnchor.constraint(equalTo: view.trailingAnchor)
-        ])
-        
-        let blurView = UIVisualEffectView.play_blurView
-        blurView.alpha = 0.0
-        view.insertSubview(blurView, belowSubview: tabBar)
-        self.blurView = blurView
-        
-        blurView.translatesAutoresizingMaskIntoConstraints = false
+        tabView.translatesAutoresizingMaskIntoConstraints = false
+        tabBarTopConstraint = tabView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor)
         NSLayoutConstraint.activate([
-            blurView.topAnchor.constraint(equalTo: tabBar.topAnchor),
-            blurView.bottomAnchor.constraint(equalTo: tabBar.bottomAnchor),
-            blurView.leadingAnchor.constraint(equalTo: tabBar.leadingAnchor),
-            blurView.trailingAnchor.constraint(equalTo: tabBar.trailingAnchor)
+            tabBarTopConstraint!,
+            tabView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            tabView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            tabView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
         ])
```

Comment:
```suggestion
            tabBarTopConstraint,
            tabView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
            tabView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
            tabView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
        ])
        self.tabBarTopConstraint = tabBarTopConstraint
```

Use declared constraint and set weak property then.